spinalcordtoolbox/centerline/nurbs.py

```diff
@@ -642,27 +642,27 @@ def reconstructGlobalApproximation(self, P_x, P_y, P_z, p, n, w):
             for k in range(m - 1):
                 somme += w[k] * self.evaluateN(Nik[i], ubar[k], u) * self.Tk(k, P_x, Nik, ubar[k], u) / denU[k]
             Tx.append(somme)
-        Tx = np.matrix(Tx)
+        Tx = np.expand_dims(np.array(Tx), 0)
```

Comment:
I was tempted to say, since we only ever want the transpose `Tx.T`, why not do `expand_dims(..., 1)` instead? But I think this code is tricky enough that it's better to replicate its workings as exactly as possible, like you did here.